app/javascript/packs/stylesheets/custom_table.scss

```diff
@@ -27,3 +27,4 @@
     opacity: 1;
     background-color: white;
 }
+
```

Comment:
I think there might be an extra line here (2 instead of 1)

Reply:
Removed here: fa73fb4